packages/web-components/src/components/footer/legal-nav.ts

```diff
@@ -45,6 +45,10 @@ class C4DLegalNav extends StableSelectorMixin(LitElement) {
   /**
    * The adjunct links container
    */
+
+  @property()
+  adjunctLinksLenght = 0;
```

Comment:
Hum, doesn't look like this gets used? If not used, lets remove all references to it.

Reply:
It is being used to trigger the update.  
The `attributeChangedCallback` function will only trigger if the property is declared.